src/dmd/e2ir.d

```diff
@@ -1363,6 +1363,32 @@ elem *toElem(Expression e, IRState *irs)
                 FuncDeclaration fd = te.var.toParent2().isFuncDeclaration();
                 assert(fd);
                 ethis = getEthis(te.loc, irs, fd);
+
+                /* Bugzilla 15984: If te is in interface contracts, ethis is a
```

Comment:
Nit, I would quote any "word" that references a symbol in the code, to avoid any confusion with a regular word that is misspelled.

Reply:
Please use:
```
https://issues.dlang.org/show_bug.cgi?id=15984
```
instead of:
```
Bugzilla 15984
```